CMakeLists.txt.user

```diff
@@ -0,0 +1,340 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You forgot to ignore this file. add it to `OTTO/.gitignore` and delete it, then commit (keep a backup though, cause its your qtcreator project file)